SnailTests/ClosureTests.swift

```diff
@@ -0,0 +1,21 @@
+//  Copyright © 2016 Compass. All rights reserved.
+
+import Foundation
+import XCTest
+@testable import Snail
+
+class ClosureTests: XCTestCase {
+    func testClosure() {
+        let expectation = self.expectation(description: "Closure Executed")
+        let disposer = Disposer()
+        let closure = Closure {
+            expectation.fulfill()
+        }.add(to: disposer)
+
+        closure.closure?()
+
+        waitForExpectations(timeout: 5, handler: nil)
```

Comment:
do we need to wait `5` seconds? It seems like this would be instant.

Reply:
yeah, I'll remove it if you're ok with the other one

Reply:
Updated this